blocks/header/header.css

```diff
@@ -41,15 +43,18 @@ header nav div:first-child {
 header nav > div:nth-child(2) {
   height: 170px;
 }
+
 header nav > div:nth-child(2) p {
-  margin: 0 0 -6px 0;
```

Comment:
do you want to remove the `margin-left` here?

Reply:
lets fix this in another PR